pkg/transaction/definitions.go

```diff
@@ -113,10 +113,11 @@ type Runtime struct {
 
 // EntryPoint - represents the entry point details for API in summary event
 type EntryPoint struct {
-	Type   string `json:"type,omitempty"`
-	Method string `json:"method,omitempty"`
-	Path   string `json:"path,omitempty"`
-	Host   string `json:"host,omitempty"`
+	Type           string `json:"type,omitempty"`
+	Method         string `json:"method,omitempty"`
+	Path           string `json:"path,omitempty"`
+	Host           string `json:"host,omitempty"`
+	UnRedactedPath string `json:"-"`
```

Comment:
Could you make the same change we did for the targetpath and resourcepath here, removing the UnRedacted* vars?

Reply:
Added the change.